view-user/src/pages/index.tsx

```diff
@@ -5,13 +5,41 @@ import Image from "next/image";
 import styles from "@/styles/Home.module.css";
 import { Header, Modal, BingoResult, Button } from "@/components/common";
 import { useRouter } from "next/router";
+import { ja } from "./locales/ja";
+import { en } from "./locales/en";
+import { MdTranslate } from "react-icons/md";
 
 const Page: NextPage = () => {
-  const [isOpened, setIsOpened] = useState(false);
+  const { locale } = useRouter()
+  const t = locale === "ja" ? ja : en;
+  const [isOpened, setIsOpened] = useState(true);
   const router = useRouter();
   const isopenBool = () => setIsOpened(!isOpened);
```

Comment:
```suggestion
  //const isopenBool = () => setIsOpened(!isOpened);
```
これ使われてないから削除がコメントアウトしておいて！

Reply:
削除しました。